docs/download.md

```diff
@@ -84,26 +84,21 @@ CUDA 11.8 through CUDA 12.0.
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
 * Verify the signature for Scala 2.12 jar:
-    `gpg --verify rapids-4-spark_2.12-24.02.0.jar.asc rapids-4-spark_2.12-24.02.0.jar`
+    `gpg --verify rapids-4-spark_2.12-24.04.0.jar.asc rapids-4-spark_2.12-24.04.0.jar`
 * Verify the signature for Scala 2.13 jar:
-    `gpg --verify rapids-4-spark_2.13-24.02.0.jar.asc rapids-4-spark_2.13-24.02.0.jar`
+    `gpg --verify rapids-4-spark_2.13-24.04.0.jar.asc rapids-4-spark_2.13-24.04.0.jar`
 
 The output of signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
-New functionality and performance improvements for this release include:
-* Discontinued support for Nvidia GPUs based on Pascal architecture.
-* Set get_json_object functionality to disabled by default.
-* Implemented string comparison in AST expressions.
-* Expanded timezone support to include options beyond UTC.
-* Optional checksums for cached files in the file cache.
-* Introduced support for Databricks 13.3 ML LTS.
-* Added support for parse_url functionality.
-* Introducing Lazy Quantifier support for regular expression functions.
-* Added support for the format_number function.
-* Enhanced batching support for row-based bounded window functions.
+* New functionality and performance improvements for this release include:
+* Enhanced decompression kernels for zstd and snappy.
+* Enhanced Parquet reading performance with modular kernels.
+* Added compatibility with Spark version 3.5.1.
+* Deprecated support for Databricks 10.4 ML LTS.
+* Added a new kernel for get_json_object, aiming to closely match Vanilla Spark output. This feature is disabled by default.
```

Comment:
Not even sure if we should mention get_json_object yet, let's see how the open 24.04 get resolved or moved to 24.06.  

```suggestion
* Added a new kernel for get_json_object, aiming to match Spark output. This feature is disabled by default.
```